{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -22,16 +23,20 @@
 @click.option('-a', '--append',
               is_flag=True,
               help='Append to the artifact instead of overwriting.')
+@click.option('-r', '--replace-keys',
```

Comment:
Seems like this will be silently ignored if you don't supply `-a`.  You could either fail out entirely if you get `-r` without `-a` or prompt the user to see if they meant to supply `-a`, or probably even just set append to true if you get `-r` args.  You need to do something though.

Reply:
Automatically appending now.